client/py/examples/control/abort/simulated_daq.py

```diff
@@ -0,0 +1,123 @@
+#  Copyright 2024 Synnax Labs, Inc.
+#
+#  Use of this software is governed by the Business Source License included in the file
+#  licenses/BSL.txt.
+#
+#  As of the Change Date specified in that file, in accordance with the Business Source
+#  License, use of this software will be governed by the Apache License, Version 2.0,
+#  included in the file licenses/APL.txt.
+
+import math
+
+import synnax as sy
+import time
+
+client = sy.Synnax()
+
+NUM_SENSOR_CHANNELS = range(0, 5)
+
+daq_time_ch = client.channels.create(
+    name="daq_time",
+    is_index=True,
+    data_type=sy.DataType.TIMESTAMP,
+    retrieve_if_name_exists=True,
+)
+
+sensor_channels = [
+    sy.Channel(
+        name=f"sensor_{i}",
+        index=daq_time_ch.key,
+        data_type=sy.DataType.FLOAT32,
+    )
+    for i in NUM_SENSOR_CHANNELS
+]
+sensor_channels = client.channels.create(sensor_channels, retrieve_if_name_exists=True)
+
+NUM_VALVE_CHANNELS = range(0, 3)
+
+valve_response_channels = client.channels.create(
+    [
+        sy.Channel(
+            name=f"valve_response_{i}",
+            index=daq_time_ch.key,
+            data_type=sy.DataType.FLOAT32,
+        )
+        for i in NUM_VALVE_CHANNELS
+    ],
+    retrieve_if_name_exists=True,
+)
+
+valve_command_time_channels = [
+    sy.Channel(
+        name=f"valve_command_{i}_time",
+        data_type=sy.DataType.TIMESTAMP,
+        is_index=True,
+    )
+    for i in NUM_VALVE_CHANNELS
+]
+
+valve_command_time_channels = client.channels.create(
+    valve_command_time_channels, retrieve_if_name_exists=True
+)
+
+valve_command_channels = [
+    sy.Channel(
+        name=f"valve_command_{i}",
+        index=valve_command_time_channels[i].key,
+        data_type=sy.DataType.FLOAT32,
+    )
+    for i in NUM_VALVE_CHANNELS
+]
+
+valve_command_channels = client.channels.create(
+    valve_command_channels,
+    retrieve_if_name_exists=True,
+)
```

Comment:
I think it would be simpler and easier to read if we only created two valve channels and one sensor channel - that's all that gets used anyways.